roles/kalite/tasks/install.yml

```diff
@@ -15,7 +15,8 @@
   package:
     name:
       - python2
-      - python-setuptools # provides setuptools-44 last version compatible with python2
+      - python-setuptools    # Provides setuptools-44 (last version compatible with python2)
+      - virtualenv           # Provides venv for python2 -- compare python3-venv in scripts/ansible
```

Comment:
For use with Ansible module: [pip](https://docs.ansible.com/ansible/latest/modules/pip_module.html) virtualenv_command: /usr/bin/virtualenv virtualenv_python: python2.7